plugins/wasmedge_opencvmini/opencvmini_func.cpp

```diff
@@ -180,5 +182,21 @@ Expect<uint32_t> WasmEdgeOpenCVMiniCvtColor::body(const Runtime::CallingFrame &,
   return Env.insertMat(Dst);
 }
 
+Expect<uint32_t> WasmEdgeOpenCVMiniResize::body(const Runtime::CallingFrame &,
+                                                uint32_t SrcMatKey,
+                                                uint32_t OutImgW,
+                                                uint32_t OutImgH,
+                                                uint32_t Interpolation) {
+  auto Src = Env.getMat(SrcMatKey);
+
+  if (!Src) {
+    return Unexpect(ErrCode::Value::HostFuncError);
+  }
+
+  cv::Mat Dst;
+  cv::resize(*Src, Dst, cv::Size(OutImgW, OutImgH), 0, 0, Interpolation);
```

Comment:
hard coded args

Reply:
Updated. I've added in the scale factors into the function definition for resize.